package.json

```diff
@@ -6,6 +6,12 @@
   "license": "MIT",
   "main": "./lib/blessed.js",
   "bin": "./bin/tput.js",
+  "scripts": {
```

Comment:
Set own `package.json` on `/docs` for the book itself, as if it were a subproject.